src/message.c

```diff
@@ -1764,30 +1769,45 @@ static void handle_domain_rule(FILE *rsp, struct token domain, char *message)
                     rule->manage = RULE_PROP_ON;
                 } else if (string_equals(value, ARGUMENT_COMMON_VAL_OFF)) {
                     rule->manage = RULE_PROP_OFF;
+                } else {
+                    bad_rule_argument_value(rsp, key, value);
+                    return;
```

Comment:
I noticed there is a mix of having an internal value in the object saying it's OK (for regex in many cases) and early return on failure. Not sure which pattern is preferred, can update this to match other pattern, just let me know.